django/applications/catmaid/static/js/widgets/treenode-table.js

```diff
@@ -461,6 +486,23 @@
       this.oTable.column(1).search(this.filter_nodetype).draw();
     }).bind(this));
 
+    // add ability to conditionally search on confidence
+    $.fn.dataTable.ext.search.push(function(settings, data, dataIndex) {
```

Comment:
This will add a new search function for each keydown event on the header. I could imagine this means asking for trouble with DataTables. Instead you could add this to `static/js/extensions.js`, where already other DataTables order and search extension live.